src/pudl/extract/vcerare.py

```diff
@@ -0,0 +1,191 @@
+"""Extract VCE Resource Adequacy Renewable Energy (RARE) generation profile data.
+
+This dataset has 1,000s of columns, so we don't want to manually specify a rename on
+import because we'll pivot these to a column. We adapt the standard extraction
+infrastructure to simply read in the data.
+
+Each annual zip folder contains a folder with three files:
+Wind_Power_140m_Offshore_county.csv
+Wind_Power_100m_Onshore_county.csv
+Fixed_SolarPV_Lat_UPV_county.csv
+
+The drive also contains one more file: vce_county_lat_long_fips_table.csv. This file is
+not partitioned, so we always read it in regardless of the partitions configured for the
+run.
+"""
+
+from collections import defaultdict
+from io import BytesIO
+
+import numpy as np
+import pandas as pd
+from dagster import AssetsDefinition, Output, asset
+
+from pudl import logging_helpers
+from pudl.extract.csv import CsvExtractor
+from pudl.extract.extractor import GenericMetadata, PartitionSelection, raw_df_factory
+
+logger = logging_helpers.get_logger(__name__)
+
+VCERARE_PAGES = [
+    "offshore_wind_power_140m",
+    "onshore_wind_power_100m",
+    "fixed_solar_pv_lat_upv",
+]
+
+
+class VCEMetadata(GenericMetadata):
+    """Special metadata class for VCE RARE renewable generation profiles."""
```

Comment:
the RE in rare stands for renewable energy so I think we can take out the word renewable here